assets/post_install_macOS.sh

```diff
@@ -15,30 +15,15 @@ logger -p 'install.info' "ℹ️ Running the custom MNE-Python post-install scri
 # Don't name the variable USER, as this one is already set.
 USER_FROM_HOMEDIR=`basename $HOME`
 
-logger -p 'install.info' "ℹ️ Fixing permissions of MNE .app bundles in ${HOME}/Applications: new owner will be ${USER_FROM_HOMEDIR}"
-chown -R $USER_FROM_HOMEDIR "${HOME}"/Applications/*\(MNE\).app
-
-logger -p 'install.info' "ℹ️ Moving MNE .app bundles from ${HOME}/Applications to /Applications/MNE-Python"
-mv "${HOME}"/Applications/*\(MNE\).app /Applications/MNE-Python/
-
-logger -p 'install.info' "ℹ️ Setting custom folder icon for /Applications/MNE-Python"
-osascript \
-    -e 'set DSTROOT to system attribute "DSTROOT"' \
-    -e 'set iconPath to DSTROOT & "/.mne-python/Menu/mne.png"' \
-    -e 'use framework "Foundation"' \
-    -e 'use framework "AppKit"' \
-    -e "set imageData to (current application's NSImage's alloc()'s initWithContentsOfFile:iconPath)" \
-    -e "(current application's NSWorkspace's sharedWorkspace()'s setIcon:imageData forFile:DSTROOT options: 0)"
```

Comment:
@jaimergp maybe this should be a new option in `conda constructor` as well: the icon to use as the macOS application.

To encapsulate all of these, should I open a new `constructor` issue mentioning that the `post_install_*.sh` scripts here should be turned into options?

Reply:
Yes, open an issue please. I might split it in several ones after analyzing the use cases.

Reply:
> Or do you want to change the icon of the actual installation directory?

yes, that's what we do here